production-infra.yaml

```diff
@@ -1,8 +1,17 @@
-deploymentName: iohk-infra
-nixopsExecutable: nixops
-nixPath: nixpkgs=https://github.com/NixOS/nixpkgs/archive/9b948ea439ddbaa26740ce35543e7e35d2aa6d18.tar.gz
-deploymentFiles:
+args:
+  nodeLimit:
```

Comment:
@domenkozar, there's a question of how much of overriding the FromJSON/ToJSON (which `yaml` rides on) that might require..